### PR TITLE
 related to #394

### PR DESCRIPTION
**Pull request by esraajb**
* add Nazareth button to landing page
* add London button to landing page
* remove Course info button from landing page

### CODE REVIEW

File: _harp/css/pages/landing/_landing-page.scss [hunk 1]
```diff
@@ -270,6 +270,30 @@
 	}
 }
 
+.programmes-info{
+	height: 10vh;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	a {
+		margin: 2%;
```
Comment (Jbarget) [inline _harp/css/pages/landing/_landing-page.scss]: just noticing the css indentation in this file is really bad. Can we keep it consistent at 2 spaces?

**Comment by esraajb** [comment]
It looks good on my Atom 
might be something with github ?

**Comment by des-des** [comment]
@esraajb no description in commit message and no description in pr

File: _harp/css/pages/landing/_landing-page.scss (hunk 1 above)
Comment (des-des) [inline _harp/css/pages/landing/_landing-page.scss]: Yeah there are tabs here. That is the problem

**Comment by des-des** [comment]
@esraajb is is because you are using tabs, use spaces instead

**Comment by esraajb** [comment]
@des-des  edited first comment

File: _harp/css/pages/landing/_landing-page.scss (hunk 1 above)
Comment (Jbarget) [inline _harp/css/pages/landing/_landing-page.scss]: how can you tell whether spaces are hard tabs or soft tabs from here?

Comment (des-des) [inline _harp/css/pages/landing/_landing-page.scss]: @Jbarget highlighting the text in the browser

Comment (msmichellegar) [inline _harp/css/pages/landing/_landing-page.scss]: GitHub renders tabs in a particular way (like this).

Comment (Jbarget) [inline _harp/css/pages/landing/_landing-page.scss]: gotcha!

**Comment by esraajb** [comment]
@des-des

**Comment by Jbarget** [comment]
sounds good @des-des

**Comment by rebeccaradding** [comment]
@esraajb @Jbarget is this done? Website not showing updates yet.

**Comment by des-des** [comment]
@esraajb we need to deploy, try the steps in the readme!

**Comment by esraajb** [comment]
@des-des am i allowed to do it or its on of the one who merged the PR?

**Comment by des-des** [comment]
I think the person who made the pull request should deploy

**Comment by esraajb** [comment]
umm ok.
followed the instruction on the readme, still no change

why we do this? why its not merged directly ?

**Comment by des-des** [comment]
@esraajb @sofer hmm I think I have had the same problem before. Cannot remember what dan said last time this happened, I think we just need to wait for caches to reset ..

I am not sure why this needs manual deployment. The hard was to do this is to build a service on Heroku that does this automatically, but there my be another way.

**Comment by rebeccaradding** [comment]
@esraajb @des-des is this still happening?

**Comment by des-des** [comment]
@rebeccaradding @esraajb did the site update in the end?

**Comment by esraajb** [comment]
yes 🎆

**Pull request merged**